scripts/components/CollectionList.js

```diff
@@ -162,26 +167,101 @@ class Table extends Component {
   }
 }
 
-function ListActions(props) {
-  const { name, onSyncClick, onResetSyncClick } = props;
+class SyncButton extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {open: false};
+  }
+
+  static strategies = {
+    [MANUAL]: {
+      label: "MANUAL resolution",
+      help: "Do not automatically resolve conflicts.",
+    },
+    [CLIENT_WINS]: {
+      label: "CLIENT_WINS resolution",
```

Comment:
I went with reusing the Kinto.js documented term, but yeah, why not, I don't know.